Foundation/NSPathUtilities.swift

```diff
@@ -344,9 +344,23 @@ extension NSString {
     }
     
     public var standardizingPath: String {
+#if os(Windows)
+        // Convert to a posix style '/' separated path
+        var converted = self as String
+        if converted.isAbsolutePath {
+            if converted.starts(with: "\\\\?\\") {
+              converted.removeFirst(3)
+            } else if !(converted.starts(with: "/") || converted.starts(with: "\\")) {
+              converted = "/" + converted
```

Comment:
Am I mistaken or does this break with UNC paths?  `\\host\C$` is not going to get mapped into a valid path I think.